renderer/features/wallet/walletSlice.js

```diff
@@ -223,6 +218,11 @@ export const connectWallet = walletId => {
     const handleMessageSubscription = proxyValue(message =>
       dispatch(receiveMessage(message))
     );
+
+    const handleInvoiceEventData = proxyValue(() => dispatch(fetchBalance()));
+
+    const handeSendPaymentEvent = proxyValue(() => dispatch(fetchBalance()));
+
```

Comment:
this is fine for now but at some point we will likely want to move these into their own receiveEventData handlers.